themes/vocabulary_theme/templates/project_list.html

```diff
@@ -52,14 +51,16 @@ <h4 class="card-title b-header"><a href="{{ repo.url }}">{{ repo.english_name }}
               </article>
             </div>
           {% endif %}
-        {% endfor %}
-      </div>
+          {% endfor %}
+        </div>
       </div>
+
       <div class="active-project">
-        <h2>All Active Projects</h2>
-        <p class="padding-vertical-big">This is a complete list of active Creative Commons software projects. Archived projects are not documented here and can be found on the <strong><a href="https://github.com/cc-archive">CC Archive GitHub organization</a>.</strong></p>
+        <h2 id="all-active-projects" aria-live="polite">All Active Projects</h2>
+        <p class="padding-vertical-big">This is a complete list of active Creative Commons software projects. Archived projects are not documented here and can be found on the <strong><a href="https://github.com/cc-archive" aria-label="Visit the CC Archive GitHub organization">CC Archive GitHub organization</a>.</strong></p>
+        
         <div class="table-container">
-          <table class="table is-bordered  is-striped">
+          <table class="table is-bordered is-striped" aria-labelledby="all-active-projects">
```

Comment:
unsure why the binding here, the `h2` precedes it and already establishes a landmark nesting relationship to the table?